editor/editor_log.cpp

```diff
@@ -273,6 +273,10 @@ void EditorLog::_undo_redo_cbk(void *p_self, const String &p_name) {
 }
 
 void EditorLog::_rebuild_log() {
+	if (messages.is_empty()) {
```

Comment:
Should this be after `clear()`?
It works this way too though.

Reply:
Didn't think it was necessary since the only way messages will realistically be empty is after a call to clear()

Reply:
I mean `log` and `messages` are separate, so *technically* it could be possible that log is outdated, but looking at the code, it can't happen currently.